tests/html/service-worker/index.html

```diff
@@ -21,22 +21,23 @@
         <button onclick="post_message_root()">Msg to Root service worker</button>
     </div>
     <script>
-        /* global navigator */
-        navigator.serviceWorker.register('iframe_sw.js', { 'scope': '/demo_iframe.html' }).then(function(registration) {
+        navigator.serviceWorker.register('/tests/html/service-worker/iframe_sw.js', { 'scope': '/tests/html/service-worker/'})
```

Comment:
This change should be reverted - it seems to be relying on having an HTTP server running from the root Servo directory, but I think we should assume a server running with tests/html/service-worker/ as the root instead.